benchmarks/bench.py

```diff
@@ -0,0 +1,24 @@
+import apache_beam as beam
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'apache_beam'](https://app.codacy.com/manual/lislemitchell/beam-sink/pullRequest?prid=5106141)